### DISCUSSION TITLE
Castellation Refinement Failed [3221225477]

### DISCUSSION THREAD

**Discussion by parisipa** [nextfoam/baram#116] [Q&A]
![image](https://github.com/user-attachments/assets/c433051c-87e9-420c-bfa3-dd621cb5af56)
Hello,

In version 24.3.3, I received this error. Any ideas why?

**Comment by kdg5424** [comment]
Hello @parisipa . This is Donggyu from NEXTFOAM.

Actually, it's hard to know why that error has happened with only that picture. I'm really sorry.
I want to help and support you but the clue is so limited.

So could you let me know more?
If there is confidential security issue, please let me know.

Best regards

Donggyu kim

**Comment by parisipa** [comment]
Hi Donggyu,

This is related also to the ticket raised here: https://github.com/nextfoam/baram/discussions/114
I have version 24.3.3 on a laptop and on the work station. On the laptop, BaramMesh and BaramFlow work perfectly, but since I need more computational power, I need the work station.

On the work station, the exact same version and installation on the work station results in neither BaramMesh nor BaramFlow working. In the meshing module, no matter what geometry I put in (simple or complex) I receive this error in castellation and cannot move past this step when geometry import, base grid etc work fine.

Equally, if  I try to solve an existing case in BaramFlow, the mesh is not read properly. Something seems to not be working on the workstation with Baram. I even tried different versions. 

Any ideas?

Thanks,
Angie

**Comment by kdg5424** [comment]
Hello @parisipa. This is Donggyu.

Could you check your mpi environment or settings? I think the reason could be the mpi setting or something like that.
Thanks

Donggyu Kim

**Reply by parisipa** [reply]
Hi Donggyu, 

Could you please be a little more specific about what I am looking for in the mpi settings that could cause a problem? Thank you

**Reply by kdg5424** [reply]
Hello @parisipa.
I mean your default mpi. Which mpi are you using now?
Could you type the below command and check it?

`mpirun -V`

Best regards

Donggyu kim

**Reply by kdg5424** [reply]
Hello @parisipa.

Could you let me know about your remote desktop? You said it's windows os. Is it cluster or just workstation?
And if it's windows, then you should check your mpi setting in control panel.
You can search it in **"Program and function"** with mpi key-word.

Thanks

Best regards

**Reply by parisipa** [reply]
Hi Donggyu,

Apologies for the delay. It is just a workstation with Windows. MPI is indeed installed 
![image](https://github.com/user-attachments/assets/f19dfce0-c1f1-4856-b41a-05beda397987)

**Reply by kdg5424** [reply]
Hello @parisipa. This is Donggyu. I think the priority of **Intel MPI** is higher than MS-MPI in your setting.
So could you delete Intel MPI and test BARAM again?

Best regards

Donggyu Kim

**Reply by parisipa** [reply]
Hi, Donggyu.

I did the test, but the problem persists even though Intel MPI is deleted. Any other ideas?

Kind regards,
Angie

**Reply by kdg5424** [reply]
Hello @parisipa. Sorry for the delay.
I recommend you to try this .exe file on cmd.
You can run this .exe file with the below code and it will show the result like the below picture.

`mpiexec -n "Your processor number" hello_mpi.exe'
[hello_mpi.zip](https://github.com/user-attachments/files/17182419/hello_mpi.zip)



![image](https://github.com/user-attachments/assets/5cc817f1-5988-416a-9a15-04f3b9253f23)
